.github/workflows/build_and_test.yml

```diff
@@ -55,6 +55,7 @@ jobs:
     runs-on: ubuntu-22.04
     env:
       GITHUB_PREV_SHA: ${{ github.event.before }}
+      PYTHON: 'python3.9'
```

Comment:
I would use a slightly different name in case it conflicts with other env names ... e.g., `PYTHON_EXECUTABLE` or `SPARK_TESTING_PYTHON`

Reply:
Got it. I agree with you. Let me use `PYTHON_TO_TEST` like `MODULES_TO_TEST`.